encodings/runend/src/primitive/compress.rs

```diff
@@ -10,15 +10,15 @@ use vortex_dtype::Nullability;
 use vortex_dtype::{match_each_integer_ptype, match_each_native_ptype, NativePType};
 use vortex_error::VortexResult;
 
-pub fn runend_encode(array: &PrimitiveArray) -> (PrimitiveArray, PrimitiveArray) {
```

Comment:
It was half leaked here.

Reply:
Yes, these are not the generic methods. However, we probably should add one. The entry point for most encoding right now is places like this https://github.com/spiraldb/vortex/blob/develop/vortex-sampling-compressor/src/compressors/runend.rs

Reply:
Are you saying have a generic fn in encodings/runend or in compressors/runend?

With signature `runend_decode/encode (array) -> array`?

Reply:
Or would you think that `RunEndArray` has can either be (end,values)  or (end, true) driven?